stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/TextViewLinkHandler.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2014-2024 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.ui.utils
+
+import android.text.Spannable
+import android.text.method.LinkMovementMethod
+import android.text.style.URLSpan
+import android.view.MotionEvent
+import android.widget.TextView
+import io.getstream.chat.android.models.User
+
+internal abstract class TextViewLinkHandler : LinkMovementMethod() {
```

Comment:
This class was previously a `public static` class in the `Utils` class, and now it changed to `internal`. From its usages it seems that such change is OK, but I still wanted to check if is OK to directly change the visibility of a public api member.

Reply:
Yes.
This class was only used internally in our XML module. It was public on the ui-common module to be accessible from XML module. After moving it to the proper module where it is used and should live, it doesn't make sense to expose it publicly, as it is not a class our customers will need on their end.

Reply:
Perfect, thank you for the explanation!